### ISSUE TITLE
Add an "edit" menu or direct link to the connected Discussion topic

### ISSUE THREAD

**Issue by expikr** [giscus/giscus#630]
The original Discussions page has a hamburger menu for basic actions such as editing and deleting, Whereas giscus is missing the UI entirely.

Would it be feasible to update the UI to reflect how it works on GitHub? Or as a stopgap measure, there could just be a direct link to the connected Discussions section on GitHub, which also have the benefit of making it quicker to moderate the section as a site owner.

**Comment by laymonage** [comment]
The timestamp is a link to the comment on GitHub. The "X comments/Y reactions" is also a link to the discussion on GitHub.

The pop-up menu is #36.

**Comment by FarisZR** [comment]
Yeah, I also couldn't find any direct link to The GitHub discussion easily.
It would be great for people who don't want to authenticate their accounts, maybe a gray button beside sign with GitHub, “comment on the discussion manually”

**Comment by ryanwwest** [comment]
> The timestamp is a link to the comment on GitHub. The "X comments/Y reactions" is also a link to the discussion on GitHub.
> 
> The pop-up menu is #36.

I'm curious how feasible it would be to have a similar link to each comment _on the blog page_. Maybe the URL path could partially match the corresponding link to take you directly to the Github discussion page, but it might be nice to allow for link sharing directly to the blog post if there's an interesting comment buried in the thread; the experience isn't quite as nice when you don't have the blog to check out on the same page in Github.

**Comment by redactedontop** [comment]
Is this happening...?
